src/viewProviders/schemas.ts

```diff
@@ -94,6 +118,15 @@ export class SchemasViewProvider implements vscode.TreeDataProvider<SchemasViewP
     return element;
   }
 
+  getParent(element: SchemasViewProviderData): SchemasViewProviderData | null {
```

Comment:
This is the magic that makes `revealSchema()` work. Well, that and each of our `SchemaTreeItem`s having completely unique `id` values.